### PR TITLE
Update en/sv labels in kbv

### PR DESCRIPTION
**Pull request by klngwll**
* remove bf2 labels
* add missing sv/en labels in kbv terms

### CODE REVIEW

File: source/vocab/details.ttl [hunk 1]
```diff
@@ -943,21 +943,21 @@
 
 :otherYear a owl:DatatypeProperty;
     sdo:domainIncludes :PrimaryProvisionActivity ;
-    rdfs:label "kompletterande datum"@sv;
+    rdfs:label "other year"@en, "kompletterande datum"@sv;
```
Comment (kwahlin) [inline source/vocab/details.ttl:946]: Should be "kompletterande årtal" in Swedish?

**Review by kwahlin — APPROVED** [review]
See comment, but looks good otherwise!

File: source/vocab/details.ttl (hunk 1 above)
Comment (klngwll) [reply source/vocab/details.ttl]: Could be! Added a note about it for now. I try to keep off content-changes which might require updated helptexts :)

Comment (kwahlin) [reply source/vocab/details.ttl]: Fair enough!

**Pull request merged**